client/packages/common/src/intl/locales/en/replenishment.json

```diff
@@ -85,11 +85,11 @@
   "label.projected": "Projected",
   "label.quantities": "Quantities",
   "label.requested-packs": "Requested packs",
-  "label.rnr-adjustments": "Adjustments",
-  "label.rnr-consumed": "Consumed",
-  "label.rnr-consumed-adjusted": "Adjusted consumption",
+  "label.rnr-adjustments": "Adjustments +/-",
+  "label.rnr-consumed": "Consumed/distributed",
+  "label.rnr-consumed-adjusted": "Adjusted consumption/distribution",
```

Comment:
Wondered whether adding a space after the `/`s so they wrap would be nicer?

i.e. this:
<img width="480" alt="Screenshot 2024-08-17 at 12 48 21 AM" src="https://github.com/user-attachments/assets/41d4e221-aa32-4e0e-b141-de1e1fa3ae49">
vs this:
<img width="346" alt="Screenshot 2024-08-17 at 12 48 28 AM" src="https://github.com/user-attachments/assets/5cea5091-4f71-4a84-8bb1-79d6b39af25c">

Not sure which we're more concerned about on tablet - seeing more columns horizontally (as working line by line) or keeping some vertical real estate, especially when keyboard open... up to you 🤷‍♀️